### PR TITLE
Remove duplicates chars in toAsci method

### PR DESCRIPTION
**Pull request by pomaxa**
subj

### CODE REVIEW

File: src/Faker/Provider/Internet.php [hunk 1]
```diff
@@ -125,13 +125,12 @@ public static function toAscii($string)
             'ო' => 'o','პ' => 'p','ჟ' => 'z','რ' => 'r','ს' => 's','ტ' => 't',
             'უ' => 'u','ფ' => 'p','ქ' => 'k','ღ' => 'g','ყ' => 'q','შ' => 's',
             'ჩ' => 'c','ც' => 't','ძ' => 'd','წ' => 't','ჭ' => 'c','ხ' => 'k',
-            'ჯ' => 'j','ჰ' => 'h','ā' => 'a','ţ' => 't','ʼ' => "'", '̧' => '',
```
Comment (fzaninotto) [inline src/Faker/Provider/Internet.php]: hard to read the diff. Can you point the duplicates?

Comment (pomaxa) [inline src/Faker/Provider/Internet.php]: 'ā' => 'a'
'ʼ' => "'"
'ḑ' => 'd'
'Ģ' => 'G'
'Š' => 'S'
'ļ' => 'l'
'ž' => 'z'
'Ē' => 'E'
'ņ' => 'n'
'Č' => 'C'

Comment (pomaxa) [inline src/Faker/Provider/Internet.php]: And I think hole this array should be protected static var, and in method ```toAscii``` there should be only logic.

**Comment by fzaninotto** [comment]
Yeah, this has been done in another PR, so yours needs rebase now.

**Comment by pomaxa** [comment]
@fzaninotto - I've done rebase and make few updates.

**Comment by fzaninotto** [comment]
Argh! needs rebase again.

**Comment by fzaninotto** [comment]
Superseded by #787

**Pull request closed without merging**